common/nymsphinx/chunking/src/fragment.rs

```diff
@@ -324,6 +338,20 @@ pub(crate) struct FragmentHeader {
 }
 
 impl FragmentHeader {
+    pub fn seed(&self) -> i32 {
+        let mut seed = self.id;
+        seed = seed.wrapping_mul(self.total_fragments as i32);
+        seed = seed.wrapping_mul(self.current_fragment as i32);
```

Comment:
I don't remember from the top of my head, but aren't we counting the fragments from 0, meaning for bunch of `FragmentHeader`, `seed()` will return 0?

Reply:
Oh crap, amazing find, I'm gonna add a `+1` in there somewhere

Reply:
```
/// ID associated with `FragmentSet` to which this particular `Fragment` belongs.
/// Its value is restricted to (0, i32::MAX].
/// Note that it *excludes* 0, but *includes* i32::MAX.
/// This allows the field to be represented using 31 bits.
```

We're good here after all